include/nbl/builtin/hlsl/workgroup/shared_clustered.hlsl

```diff
@@ -0,0 +1,9 @@
+// Copyright (C) 2023 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+#ifndef _NBL_BUILTIN_HLSL_WORKGROUP_SHARED_CLUSTERED_INCLUDED_
+#define _NBL_BUILTIN_HLSL_WORKGROUP_SHARED_CLUSTERED_INCLUDED_
+
+#include "nbl/builtin/hlsl/workgroup/shared_ballot.hlsl"
+
+#endif
```

Comment:
get rid of this header, we don't have clustered ops yet and clustered ops will probably use same mount of smem as non clustered arithmetic ops